buildSrc/src/main/groovy/nva.publication.api.git-hooks.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'base'
+}
+
+def scriptsPrePushPath = file('.scripts/pre-push')
+def prePushHookPath = file('./.git/hooks/pre-push')
+
+tasks.register('copyPrePushHook', Copy) {
+    from scriptsPrePushPath
+    into prePushHookPath.parentFile
+    onlyIf {
+        !prePushHookPath.exists() || scriptsPrePushPath.text != prePushHookPath.text
+    }
+    doLast {
+        if (!System.properties['os.name'].toLowerCase().contains('windows')) {
+            println "Making ${prePushHookPath} executable"
+            if (!prePushHookPath.setExecutable(true)) {
+                throw new GradleException("Failed to make ${prePushHookPath} executable.")
+            }
+        }
+    }
+}
+
+build {
```

Comment:
Is there a more appropriate task on which to enforce this?

Reply:
could maybe have it on git push so to not affect build time

Reply:
ah wait, it is pre-push already. do we need this?